README.md

```diff
@@ -10,12 +10,12 @@
 
 There are [four main combinators in the `Promise` landscape](https://v8.dev/features/promise-combinators).
 
-| name                 | description                                     |                                                                             |
-| -------------------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
-| `Promise.allSettled` | does not short-circuit                          | [separate proposal](https://github.com/tc39/proposal-promise-allSettled) 🔜 |
-| `Promise.all`        | short-circuits when an input value is rejected  | added in ES2015 ✅                                                          |
-| `Promise.race`       | short-circuits when an input value is settled   | added in ES2015 ✅                                                          |
-| `Promise.any`        | short-circuits when an input value is fulfilled | this proposal 🆕                                                            |
+| name                 | description                                     |                    |
+| -------------------- | ----------------------------------------------- | ------------------ |
+| `Promise.allSettled` | does not short-circuit                          | added in ES2015 ✅ |
```

Comment:
It would be scheduled for inclusion in ES2020 (but depending on when the PR actually gets merged and approved). Definitely not ES2015.

Reply:
Let's still link to the proposal until this is actually merged upstream.

Reply:
Ok sure